src/Evolution/DgSubcell/CMakeLists.txt

```diff
@@ -12,13 +12,15 @@ spectre_target_headers(
   ActiveGrid.hpp
   DgSubcell.hpp
   Matrices.hpp
+  Mesh.hpp
```

Comment:
[thinking out loud, not a change request] Do you think it could be helpful to name these file `FdMesh` or `CellCenteredMesh` or similar, to make obvious how this is different from the `Mesh` in the `Spectral` dir?

Reply:
I guess I prefer files having the same name as the thing they have in it, which is a function called `mesh` :shrug: (also, filename renames are a pain for rebases...)

Reply:
I agree, our policy is to put names into namespaces and subdirectories rather than make names longer...